### PR TITLE
CHEAP-714 Auto mutation code generation

### PR DESCRIPTION
**Pull request by junzhengca**
This PR adds a mutation code generation script that can be used by calling

```
npm run add:mutation Object.methodName
```

The script will first check if a controller called `ObjectController` exists, if not, it will create that controller and add all correct imports to `App.js`

Then the script will create `methodName` mutation method, with all bindings to `App.js` as well.

### CODE REVIEW

**Comment by ralph-dev** [comment]
This is not on roadmap ... <.< 
Why you working on non pilot V2 stuff lol (Why not app stuff :O)

File: src/scripts/addMutation.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+async function ensureControllerExists(name) {
+    const controllerFilePath = `./src/app/controllers/${name}Controller.js`;
+    const appFilePath = `./src/app/App.js`;
+    try {
+        await fs.access(controllerFilePath);
+    } catch (e) {
+        console.log("Controller does not exist.");
+        await fs.writeFile(controllerFilePath, `class ${name}Controller {
+    constructor(app) {
```
Comment (ralph-dev) [inline src/scripts/addMutation.js:25]: This identation though :O

**Comment by junzhengca** [comment]
@ralph-dev because either this or nothing haha. Can't risk doing app when watching psy lecture.

File: src/scripts/addMutation.js (hunk 1 above)
Comment (junzhengca) [reply src/scripts/addMutation.js:25]: ?

Comment (ralph-dev) [reply src/scripts/addMutation.js:25]: Looks like the ` ` is misaligned on github

**Review by ralph-dev — APPROVED** [review]
LGTM

**Pull request merged**